todo_app/test_ViewModel.py

```diff
@@ -0,0 +1,53 @@
+import pytest
+
+from todo_app.ViewModel import ViewModel
+
+from todo_app.todoItem import ToDoItem
+
+from datetime import datetime, timedelta
+
+def test_todo_items():
+    items = []
+    items.append(ToDoItem(1, 'List saved todo items', '', None, None, datetime.today().strftime("%Y-%m-%dT%H:%M:%S.%fZ"), 'To Do'))
+    view_model = ViewModel(items)
+    todo_items = view_model.todo_items
+    assert len(todo_items) == 1
+
+def test_doing_items():
+    items = []
+    items.append(ToDoItem(1, 'List saved todo items', '', None, None, datetime.today().strftime("%Y-%m-%dT%H:%M:%S.%fZ"), 'Doing'))
+    view_model = ViewModel(items)
+    doing_items = view_model.doing_items
+    assert len(doing_items) == 1
+
+def test_done_items():
+    items = []
+    items.append(ToDoItem(1, 'List saved todo items', '', None, None, datetime.today().strftime("%Y-%m-%dT%H:%M:%S.%fZ"), 'Done'))
+    view_model = ViewModel(items)
+    done_items = view_model.done_items
+    assert len(done_items) == 1
+
+def test_recent_done_items():
+    items = []
+    items.append(ToDoItem(1, 'List saved todo items', '', None, None, datetime.today().strftime("%Y-%m-%dT%H:%M:%S.%fZ"), 'Done'))
+    view_model = ViewModel(items)
+    done_items = view_model.recent_done_items
+    assert len(done_items) == 1
+
+def test_older_done_items():
+    items = []
+    items.append(ToDoItem(1, 'List saved todo items', '', None, None,  datetime.strftime((datetime.today()-timedelta(days=1)), "%Y-%m-%dT%H:%M:%S.%fZ"), 'Done'))
+    view_model = ViewModel(items)
+    done_items = view_model.older_done_items
+    assert len(done_items) == 1
```

Comment:
Your tests could be improved - all of these tests would pass even if ViewModel's methods didn't do any filtering, because you only add one item in each test.